### PR TITLE
Added Mobible App metrics

### PR DESCRIPTION
**Pull request by sophiaxu0424**
- Added Mobible App metrics
link to cards: https://instana.kanbanize.com/ctrl_board/206/cards/137392/details/

### CODE REVIEW

**Review by schmidtl — APPROVED** [review]
code looks good to me, did not yet run it locally.

**Comment by sophiaxu0424** [comment]
Team EUM is creating new API call for /api/mobile-app-monitoring/catalog/tags for getting all available mobile app metrics data. Still need to wait for its done and tested this PR before get merged.

Card link: https://instana.kanbanize.com/ctrl_board/53/cards/138566/details/

**Pull request merged**
